README.md

```diff
@@ -227,6 +231,23 @@ Commands:
 A controlled resource is a cloud resource managed by the Terra workspace on behalf of the user.
 Currently, the only supported controlled resource is a bucket.
 
+#### Data References
+```
+Usage: terra data-refs [COMMAND]
+Reference data in the workspace context.
+Commands:
+  list          List all data references.
+  add           Add a new data reference.
+  delete        Delete an existing data reference.
+  check-access  Check if you have access to a data reference.
+  resolve       Resolve a data reference to its cloud id or path.
+```
+
+A data reference points to a cloud resource that the user wants to read or write data to.
+Data references include some types of controlled resources (e.g. bucket but not VM) and 
```

Comment:
nit: maybe note that it's about data based controlled resources, not compute based controlled resources?

Reply:
done.